ui/src/app/settings/components/ProxyForm/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./ProxyForm";
```

Comment:
These seems like a strange pattern to me. I know it's other places in the app, but why not just export and import from the component directly?

Reply:
Yeah it is a bit weird. I remember trying to do something similar in vanilla react to get tree-shaking to work properly (which it never did), but that was a lib and this is an app. Maybe it makes sense if there are multiple related, exportable files within the same folder, e.g.

#### `exporter.js`
``` javascript
export { Title } from "./Title";
export { Body } from "./Body";
export { Footer } from "./Footer";
```

#### `importer.js`
``` javascript
import { Title, Body, Footer } from "./exporter.js"

Reply:
@steverydz because then your import looks like import { Table } from 'components/Table/Table'